packages/from-query/src/generate.ts

```diff
@@ -110,8 +123,15 @@ export default (OPTIONS: IFromQueryOptions): (ir: IOperation) => string => {
     private buildDeclaration (selection: Selection): string {
       switch (selection.kind) {
         case 'Field':
+          const foo = !!selection.directives['include'] || !!selection.directives['skip']
```

Comment:
can we find a better name than `foo` 😄

Reply:
can we reuse the `hasDirectives` above? we can then do it inline below

```ts
{
  // ...
  optional: hasDirectives(selection.directives)
}
```

Reply:
ooh good catch, got excited that I got it working and missed this cleanup spot lol